### ISSUE TITLE
Audio session release

### ISSUE THREAD

**Issue by jonitef** [hyochan/react-native-audio-recorder-player#344]
Please add when recording/playing audio is paused/stop:
`AVAudioSession.sharedInstance().setActive(false, options: .notifyOthersOnDeactivation)`

So other audio sources e.g. Spotify can continued playing in background.

**Comment by hyochan** [comment]
@jonitef Nice issue! Could you kindly open a `PR` for this?

**Comment by jonitef** [comment]
Hi! Thanks for the answer. Right now I'm so busy with my job that I unfortunately have no time :/

**Comment by hyochan** [comment]
> Hi! Thanks for the answer. Right now I'm so busy with my job that I unfortunately have no time :/

Understood. Same here :cry:

**Comment by kaiykwok2** [comment]
i think this is a very critical feature!

**Comment by hyochan** [comment]
Good catch — releasing the AVAudioSession with `setActive(false)` after recording/playback stops is best practice so other apps (e.g. music players) can resume. This is a clean iOS enhancement. Tracking it.

> 🤖 AI-native maintenance. See #787 for context.
